lib/daimon_markdown/plugins/toc_pages.rb

```diff
@@ -0,0 +1,101 @@
+module DaimonMarkdown
+  class Plugin
+    class TableOfContentsPages < Base
+      Plugin.register("toc_pages", self)
+
+      include MarkdownHelper
+
+      def call(limit = nil)
+        return unless context.key?(:request)
+
+        request = context[:request]
+
+        current_page = if request.params.key?(:page)
```

Comment:
How about passing context expressly (`:page`, `:public_id` etc) instead of `:request`?
It is difficult for me to find what params this plugin needs 😕